include/ppx/knob.h

```diff
@@ -314,6 +317,75 @@ class KnobDropdown final
     std::vector<T> mChoices;
 };
 
+// KnobFlag is intended for parameters that cannot be adjusted when the application is run
+// They will not be displayed in the UI and cannot have their values changed by the application
```

Comment:
It is currently possible to display those in UI with SetVisible(true). I think that's fine and maybe could have uses, but if we keep that behaviour, we should not call out that they will not be displayed in the UI, perhaps only that they are hidden by default. Alternatively we don't allow them to be visible and leave this comment as is.